frontend/src/pages/Projects/Project/ChangeRequest/FormConfig.ts

```diff
@@ -104,7 +103,6 @@ export const FormConfig = (query: UseQueryResult<AxiosResponse, unknown>) => {
       fieldType: "multiText",
       fieldLabel: "Types",
       width: "full",
-      // required: true,
```

Comment:
We might lose track of these if we remove them. It seems like these fields are supposed to be required but it's disabled for now for whatever reason.

Reply:
Types can be left commented out b/c it isn't in the addUpdateBody validator, so it's not required currently in the model.

`cr_contact` is in the addUpdateBody, but not in the getAll() validator. 

It looks like there's some incomplete data modelling behind this, and that is the most likely cause of Adam's TODO note saying it's broken.

The app allows me to submit a record without specifying the Types or the  without both of these fields, so I think I'll leave the comments in for now.
@ShawnTurple ^^

Reply:
Ok great. let's handle that in the next pr i guess?